src/RemoteInboxNotifications/OptionRuleProcessor.php

```diff
@@ -42,7 +42,7 @@ public function process( $rule, $stored_state ) {
 		}
 
 		if ( isset( $rule->transformers ) && is_array( $rule->transformers ) ) {
-			$option_value = TransformerService::apply( $option_value, $rule->transformers, $rule->default );
+			$option_value = TransformerService::apply( $option_value, $rule->transformers, $default );
 		}
```

Comment:
A `$default` value is already being set on line 26, that checks if `$rule->default` isset.